docs/en/observability/apm/command-reference.asciidoc

```diff
@@ -625,250 +358,50 @@ ifndef::serverless[]
 
 ["source","sh",subs="attributes"]
 -----
-{beatname_lc} run [FLAGS]
+apm-server run [FLAGS]
 -----
 
 Or:
 
 ["source","sh",subs="attributes"]
 -----
-{beatname_lc} [FLAGS]
+apm-server [FLAGS]
 -----
 
 *FLAGS*
 
-ifeval::["{beatname_lc}"=="packetbeat"]
-*`-I, --I FILE`*::
-Reads packet data from the specified file instead of reading packets from the
-network. This option is useful only for testing {beatname_uc}.
-+
-["source","sh",subs="attributes"]
------
-{beatname_lc} run -I ~/pcaps/network_traffic.pcap
------
-endif::[]
-
 *`-N, --N`*:: Disables publishing for testing purposes.
 
-ifeval::["{beatname_lc}"=="packetbeat"]
-*`-O, --O`*::
-Read packets one by one by pressing _Enter_ after each. This option is useful
-only for testing {beatname_uc}.
-endif::[]
-
 *`--cpuprofile FILE`*::
 Writes CPU profile data to the specified file. This option is useful for
-troubleshooting {beatname_uc}.
-
-ifeval::["{beatname_lc}"=="packetbeat"]
-*`-devices`*::
-Prints the list of devices that are available for sniffing and then exits.
-endif::[]
-
-ifeval::["{beatname_lc}"=="packetbeat"]
-*`-dump FILE`*::
-Writes all captured packets to the specified file. This option is useful for
-troubleshooting {beatname_uc}.
-endif::[]
+troubleshooting APM Server.
 
 *`-h, --help`*::
 Shows help for the `run` command.
 
 *`--httpprof [HOST]:PORT`*::
 Starts an HTTP server for profiling. This option is useful for troubleshooting
-and profiling {beatname_uc}.
-
-ifeval::["{beatname_lc}"=="packetbeat"]
-*`-l N`*::
-Reads the pcap file `N` number of times. The default is 1. Use this option in
-combination with the `-I` option. For an infinite loop, use _0_. The `-l`
-option is useful only for testing {beatname_uc}.
-endif::[]
-
-*`--memprofile FILE`*::
-Writes memory profile data to the specified output file. This option is useful
-for troubleshooting {beatname_uc}.
```

Comment:
These three lines were accidentally deleted.

Reply:
FWIW, I noticed this because the HTML diff showed a change:
```
+0 -9 [en/observability/master/command-line-options.html](https://observability-docs_bk_3673.docs-preview.app.elstc.co/guide/en/observability/master/command-line-options.html)
```

Diffing the built-docs [source](https://github.com/elastic/built-docs/blob/observability-docs_bk_3673/html/en/observability/master/command-line-options.html) led to finding this missing in the PR preview:  

```
<span class="strong strong"><strong><code class="literal">--memprofile FILE</code></strong></span>
</span>
</dt>
<dd>
Writes memory profile data to the specified output file. This option is useful
for troubleshooting APM Server.
</dd>
<dt>
<span class="term">
```